src/input/mod.rs

```diff
@@ -205,6 +205,7 @@ pub fn add_seat(
             .expect("Failed to load xkb configuration files");
     }
     seat.add_pointer();
+    seat.add_touch();
```

Comment:
I am not sure we want to do that unconditionally or only, if we actually detect a touch device.

The problem with that is obviously, that I am not sure any clients bind this dynamically...

Reply:
Yeah, it's trivial to add when we get a touch device, and not too hard to track how many touch devices we have and remove it when there aren't any more. But if clients don't handle that properly with keyboard/pointer, I expect that's an issue with touch too.

Reply:
Likely a safe assumption..